dwave_networkx/drawing/chimera_layout.py

```diff
@@ -23,8 +23,8 @@
 from networkx import draw
```

Comment:
Looks like you need to add `# coding=utf-8` to the top of this file to fix the tests in python2. See https://stackoverflow.com/questions/10589620/syntaxerror-non-ascii-character-xa3-in-file-when-function-returns-%C2%A3

Reply:
Is this something you'll be needing in every file?
I'm guessing this is because of the description of matplotlib's scatter.
Is it enough to just remove the characters that cause this?

Reply:
Yes, removing the characters would do it. It looks like the problem is actually in `dwave_networkx/drawing/qubit_layout.py`

Reply:
OK, I'll make the changes and make sure it passes. I don't want to add the encoding line if it's not necessary.